src/lib.rs

```diff
@@ -250,18 +230,18 @@ impl OnlineWallet {
         };
         let database = AnyDatabase::from_config(&any_database_config)?;
         let blockchain = AnyBlockchain::from_config(&any_blockchain_config)?;
-        let wallet = Mutex::new(Wallet::new(
+        let _wallet = Mutex::new(BdkWallet::new(
```

Comment:
usually the _ is used for variables that aren't used, so I think this should be renamed back to `wallet` or possibly `wallet_mutex` if you want to be more descriptive.